### PR TITLE
 Start working on tiles

### PR DESCRIPTION
**Pull request by ilia280**

### CODE REVIEW

**Comment by ilia280** [comment]
@dieselT @BggClr could you please review?

At this moment available only the custom tile. News and event tiles will be added after news and events will be created

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # components_umbraco_tiles_block
-Umbraco tiles block
+Umbraco tiles block for Base Section 
+
+Nuget:
+* Install-Package Skybrud.Umbraco.GridData
+* Install-Package UmbracoLinkPicker
+
+Dependencies:
+* [Base Section](https://github.com/graphuk/components_umbraco_base_section) (install it via Stamp)
```
Comment (BggClr) [inline README.md]: Here should be command for stamp

Comment (graphuk) [reply README.md]: Added

File: GridConverters/TilesJsonConverter.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+					return tile.ToObject<GridControlCustomTile>(serializer);
+				default:
+					return null;
+			}
+		}
+
+		private static object MapToNewsTile(JObject tile)
+		{
+			var contentId = tile.Value<int>("contentId");
+			if (contentId > 0)
```
Comment (dieselT) [inline GridConverters/TilesJsonConverter.cs]: No need to check `contentId` because it is required on UI

Comment (ilia280) [reply GridConverters/TilesJsonConverter.cs]: @dieselT done

**Pull request merged**